src/tetris.py

```diff
@@ -1,4 +1,10 @@
-from shape import Shape
+from shape import Shape, create_shape
+import random
+
+red = (255, 0, 0)
+blue = (0, 0, 255)
+yellow = (255, 255, 0)
+white = (0, 0, 0)
```

Comment:
Bu tarz global ve degismeyecek olan degiskenleri buyuk harflerle yazabiliriz
```suggestion
RED = (255, 0, 0)
BLUE = (0, 0, 255)
YELLOW = (255, 255, 0)
WHITE = (0, 0, 0)
```